README.md

```diff
@@ -0,0 +1,60 @@
+# TechJobs, MVC Edition
+
+For [this assignment](https://education.launchcode.org/java-web-development/assignments/tech-jobs-mvc.html) students are asked to once again refactor a TechJobs application, this time using an MVC 
+design pattern.
+
+## Assignment Requirements
+
+1. Review Starter Code
+2. Complete the List Views
+3. Complete the `searchController`
+4. Display Search Results
+
+## Unit Tests
+
+The starter code contains 4 unit test classes.  As with previous graded assignments, each test class was created for a specific task in the instructions.
+
+`TestTaskTwo` verifies the work done to complete the List Views.  This includes displaying a list of jobs and adding a View All link.
+
+`TestTaskThree` verifies work done to complete the `SearchController`.  This includes verifying how the `displaySearchResults` method was created and making sure that the moths calls both `findAll` and `findByColumnAndValue` methods.  This test does not check behavior.
+
+`TestTaskFour` verifies the behaviors required for `displaySearchResults` to pass data to a view.  
+
+`ApplicationTest` verifies that the application is able to run.
+ 
+### Appearance and Code Check:
+ 
+[Working Demo App](https://lc-java-3-demo.herokuapp.com/)
```

Comment:
Update link -> still pointing to heroku

Reply:
Please add verbiage about why students may run into issues with the app.

Reply:
Done!